idaes/generic_models/unit_models/column_models/column.py

```diff
@@ -0,0 +1,1144 @@
+                return blk.interphase_mass_transfer[t, x, j] == (
+                    blk.k_v[t, x, j] *
+                    blk.area_interfacial[t, x] * blk.area_column *
+                    (blk.vapor_phase.properties[t, x].mole_frac_comp[j] *
+                     blk.vapor_phase.properties[t, x].pressure -
+                     blk.pressure_equil[t, x, j]))
+            else:
+                return blk.interphase_mass_transfer[t, x, j] == 0.0
+
+        self.mass_transfer_vapor = Constraint(self.flowsheet().time,
+                                        self.vapor_phase.length_domain,
+                                        liq_comp,
+                                        rule=rule_mass_transfer,
+                                        doc="mass transfer in vapor phase")
+
+        # Liquid phase mass transfer handle
+        @self.Constraint(self.flowsheet().time,
+                          self.liquid_phase.length_domain,
+                          self.liquid_phase.properties.phase_component_set,
+                          doc="mass transfer to liquid")
+        def liquid_phase_mass_transfer_handle(blk, t, x, p, j):
+            if x == self.liquid_phase.length_domain.last():
+                return blk.liquid_phase.mass_transfer_term[t, x, p, j] == 0.0
+            else:
+                zf = self.liquid_phase.length_domain.at(self.zi[x].value + 1)
+                if j in solute_comp_list:
+                    return blk.liquid_phase.mass_transfer_term[t, x, p, j] == \
+                        blk.interphase_mass_transfer[t, zf, j]
+                else:
+                    return blk.liquid_phase.mass_transfer_term[t, x, p, j] == \
+                        0.0
+
+        # Vapor phase mass transfer handle
+        @self.Constraint(self.flowsheet().time,
+                         self.vapor_phase.length_domain,
+                         self.vapor_phase.properties.phase_component_set,
+                         doc="mass transfer from vapor")
+        def vapor_phase_mass_transfer_handle(blk, t, x, p, j):
+            if x == self.vapor_phase.length_domain.first():
+                return blk.vapor_phase.mass_transfer_term[t, x, p, j] == 0.0
+            else:
+                if j in equilibrium_comp:
+                    return blk.vapor_phase.mass_transfer_term[t, x, p, j] == \
+                        -blk.interphase_mass_transfer[t, x, j]
+                else:
+                    return blk.vapor_phase.mass_transfer_term[t, x, p, j] == \
+                        0.0
+
+        # Heat transfer coefficients
+        # Vapor-liquid heat transfer coefficient [J/m2.s.K]
+
+        self.h_v = Var(self.flowsheet().time,
+                       self.vapor_phase.length_domain,
+                       initialize=100,
+                       doc='''Vapor-liquid heat transfer coefficient''')
+
+        # Vapor-liquid heat transfer coeff modified by Ackmann factor [J/m.s.K]
+        def rule_heat_transfer_coeff_Ack(blk, t, x):
```

Comment:
Similarly, I think the generic model should just have `Q=U*A_inter*deltaT` - definition of `U` is something for more detailed models.